standard-cluster-services/src/main/java/org/terracotta/entity/IEntityMessenger.java

```diff
@@ -39,6 +39,18 @@
   void messageSelf(EntityMessage message) throws MessageCodecException;
 
   /**
+   * Defer a message based on another message, which will be sent in the future.
+   *
+   * @param tag String reason tag, mainly for debugging.
```

Comment:
So can we change "mainly for debugging" to "for debugging", or "for debugging and reporting"?